x-pack/plugins/triggers_actions_ui/public/application/sections/alert_form/alert_add.tsx

```diff
@@ -89,16 +88,19 @@ const AlertAdd = ({
     setAlertProperty('alertTypeId', alertTypeId ?? null);
   }, [alertTypeId]);
 
-  const closeFlyout = useCallback(() => {
-    setAlert(initialAlert);
```

Comment:
We no longer need to reset the alert to an initial alert on close because with [this PR](https://github.com/elastic/kibana/pull/85462) we are removing the flyout on close completely instead of just hiding it.